students/brandon_nguyen/session06/mailroom4.py

```diff
@@ -65,21 +69,77 @@ def create_rpt():
     print("\n\n")
 
 
-def sort_sum():
+# Test1 DONE
+def sort_sum4_report(dict_db):
     # creating a new list with computed value for easy printing.
     """
-    This function returned a sorted list of by on order amount3
+    This function returned a sorted list of by on order amount
     """
     dblist = []
     dbSum = []
     # quick way to convert back to list to reuse existing code.
     # list comprehension
-    [dblist.append((i, j)) for i, j in donor_db.items()]
+    [dblist.append((i, j)) for i, j in dict_db.items()]
```

Comment:
Still not very desirable list comprehension, though it'll work. Desired style is:
```
dblist = [(i, j) for i, j in dict_db.items()]
```
In fact, in this case, probably `dblist = list(db.items())` would be enough.

Same for the next line.